tools/ci/after_install.sh

```diff
@@ -12,3 +12,22 @@ if [ "$TEST_SUITE" = "spec:javascript" ]; then
   yarn list
   echo "travis_fold:end:YARN_LOCK"
 fi
+
+if [ "$TEST_SUITE" = "spec" ]; then
```

Comment:
also needs a "is this a PR" check, so as not to compare master with master
and a base branch check, so that it doesn't fail for [IVANCHUK] PRs

Reply:
TOOD `"$TRAVIS_PULL_REQUEST" && "$TRAVIS_TAG" = ""`